app/webpacker/src/javascripts/help_popup.js

```diff
@@ -0,0 +1,49 @@
+$("#close_box").on("click", function(event) {
+  hide_help_popup();
+})
+
+$("#close_box_longer").on("click", function(event) {
+  setCookie("show_digital_assistant_popup", "disable", 28);
+  hide_help_popup();
+})
+
+function show_help_popup() {
+  $("#help_popup").fadeIn(1000);
+}
+
+function hide_help_popup() {
+  $("#help_popup").css("display", "none");
+}
+
+setTimeout(checkCookieAndBrowser, 30000);
```

Comment:
I am not sure about the naming `checkCookieAndBrowser`,
this is also showing the popup, maybe we could rename:

checkCookieAndBrowser -> show_help_popup
and the 
show_help_popup -> fadein_help_popup

just an idea